Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.yml

```
@@ -0,0 +1,220 @@
+      name: kind
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: location
+      required: true
+      secret: false
+    - default: false
+      isArray: true
+      name: tags
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: custom_domain_name
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: use_sub_domain_name
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: enc_key_source
+      predefined:
+      - Microsoft.Storage
+      - Microsoft.Keyvault
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: enc_requireInfrastructureEncryption
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: enc_keyvault_key_name
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: enc_keyvault_key_version
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: enc_keyvault_uri
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: access_tier
+      predefined:
+      - Hot
+      - Cool
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: supports_https_traffic_only
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: is_hns_enabled
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: large_file_shares_state
+      predefined:
+      - Disabled
+      - Enabled
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: allow_blob_public_access
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: minimum_tls_version
+      predefined:
+      - TLS1_0
+      - TLS1_1
+      - TLS1_2
+      required: false
+      secret: false
+    deprecated: false
+    execution: false
+    name: azure-storage-account-create-update
+  - arguments:
+    - default: false
+      isArray: false
+      name: account_name
+      required: true
+      secret: false
+    deprecated: false
+    execution: false
+    name: azure-storage-blob-service-properties-get
+  - deprecated: false
+    execution: false
+    name: azure_storage_blob_service_properties_set
```

```suggestion
    name: azure-storage-blob-service-properties-set
```

I know you are yet to implement this but just note this.